BrainPortal/app/models/data_provider.rb

```diff
@@ -1572,7 +1572,7 @@ def is_excluded?(pathname) #:nodoc:
   #
   #   'Mike O'\''Connor'
   def shell_escape(s) #:nodoc:
-    s.to_s.bash_escape(true)  # in config/initializers/core_extentions/string.rb
+    s.to_s.bash_escape(true)  # in config/initializers/core_extensions/string.rb
```

Comment:
Actually this comment is very old and the path of the file has changed completely, so we might as well adjust the whole thing. The line should be

```ruby
    s.to_s.bash_escape(true)  # lib/cbrain_extensions/string_extensions/utilities.rb
```